caas/kubernetes/provider/bootstrap.go

```diff
@@ -1348,6 +1392,17 @@ func (c *controllerStack) controllerContainers(jujudCmd, controllerImage string)
 				SubPath:   cloudconfig.FileNameBootstrapParams,
 				ReadOnly:  true,
 			},
+			{
+				Name:      application.CharmVolumeName,
+				MountPath: "/charm/bin/pebble",
+				SubPath:   "charm/bin/pebble",
+				ReadOnly:  true,
+			},
+			//{
+			//	Name:      application.CharmVolumeName,
```

Comment:
We'll need this for the charm container to have access to the api-server pebble instance.